atramhasis/__init__.py

```diff
@@ -44,9 +44,4 @@ def main(global_config, **settings):
 
     config.include('atramhasis.data:db')
 
-    # if standalone include skos sample data
```

Comment:
Currenlty unable to run in standalone mode. I switched back to sqlite, upgraded alembic and then ran pserve development.in. Error:

```
2020-02-21 09:33:16,374 ERROR [atramhasis.views.exception_views][waitress] No SKOS registry found, please check your application setup
Traceback (most recent call last):
  File "/home/koen/.virtualenvs/atramhasis/lib/python3.6/site-packages/pyramid/tweens.py", line 41, in excview_tween
    response = handler(request)
  File "/home/koen/.virtualenvs/atramhasis/lib/python3.6/site-packages/pyramid/router.py", line 148, in handle_request
    registry, request, context, context_iface, view_name
  File "/home/koen/.virtualenvs/atramhasis/lib/python3.6/site-packages/pyramid/view.py", line 667, in _call_view
    response = view_callable(context, request)
  File "/home/koen/.virtualenvs/atramhasis/lib/python3.6/site-packages/pyramid/config/views.py", line 188, in attr_view
    return view(context, request)
  File "/home/koen/.virtualenvs/atramhasis/lib/python3.6/site-packages/pyramid/config/views.py", line 214, in predicate_wrapper
    return view(context, request)
  File "/home/koen/.virtualenvs/atramhasis/lib/python3.6/site-packages/pyramid/viewderivers.py", line 401, in viewresult_to_response
    result = view(context, request)
  File "/home/koen/.virtualenvs/atramhasis/lib/python3.6/site-packages/pyramid/viewderivers.py", line 111, in _class_requestonly_view
    inst = view(request)
  File "/home/koen/Projecten/python/skos/atramhasis/atramhasis/views/views.py", line 44, in __init__
    raise SkosRegistryNotFoundException()
atramhasis.errors.SkosRegistryNotFoundException: 'No SKOS registry found, please check your application setup'

```

Reply:
The true error is actually hidden because of  
```
if hasattr(request, 'skos_registry')
```
There, the factory will run, but `AttributeError` will be swallowed. That's most likely what happened. I'll add code to log AttributeErrors in the factory.

The most likely reason for an AttributeError is that you have to run `initialize_atramhasis_db development.ini`, but without the trace i can't tell for sure.

Reply:
Ok, dat was inderdaad nog nodig. Ben de readthedocs ondertussen ook terug aan het aanpassen zodat die terug builden.

Wel vastgesteld dat de toepassing zeer traag is en momenteel een aantal frontend files mist. Maar dat kan opgelost worden nadat de testen terug werken in een andere branch.

Reply:
Bij het laden van de homepage gaat het even trager door
```
GET request to http://vocab.getty.edu/aat/.rdf
GET request to http://vocab.getty.edu/aat/.rdf
GET request to http://vocab.getty.edu/tgn/.rdf
GET request to http://vocab.getty.edu/tgn/.rdf
GET request to http://purl.org/heritagedata/schemes/eh_period.rdf
GET request to http://purl.org/heritagedata/schemes/eh_period.rdf
GET request to http://purl.org/heritagedata/schemes/eh_tmt2.rdf
GET request to http://purl.org/heritagedata/schemes/eh_tmt2.rdf
GET request to http://purl.org/heritagedata/schemes/eh_tbm.rdf
GET request to http://purl.org/heritagedata/schemes/eh_tbm.rdf
```